sources/packages/backend/libs/integrations/src/cra-integration/cra-income-verification.processing.service.ts

```diff
@@ -198,7 +198,10 @@ export class CRAIncomeVerificationProcessingService {
   async processResponses(): Promise<ProcessSftpResponseResult[]> {
     const remoteFilePaths = await this.craService.getResponseFilesFullPath(
       this.ftpResponseFolder,
-      /CCRA_RESPONSE_[\w]*\.txt/i,
+      new RegExp(
+        `[${this.configService.craIntegration.environmentCode}]BCSA\\d{5}\\.TXT`,
```

Comment:
BCSA here does not represent the CRA area code from `this.configService.craIntegration.programAreaCode`?

Reply:
There is no confirmation at this moment that programAreaCode should be used, hence moving ahead with the hardcoded value for the hotfix seems to be ok.

Reply:
Can we remove the remove `CRA_PROGRAM_AREA_CODE` from the release instruction then?